### ISSUE TITLE
Loss computation

### ISSUE THREAD

**Issue by 2006pmach** [DrSleep/tensorflow-deeplab-resnet#156]
In the file train.py I am wondering why the computation of `label_proc` uses the static size of the `raw_output` with `.get_size()` and not the actual dynamic size with `tf.size()`. I checked and observed that the static size on Pascla VOC with 321,321 as input size results in 41,41. Any reasons why you do that? Can we not drop this line completely and calculate the loss on the full image resolution?

The line I am referring to is line 168
`label_proc = prepare_label(label_batch, tf.stack(raw_output.get_shape()[1:3]), num_classes=args.num_classes, one_hot=False)`

**Comment by 2006pmach** [comment]
I just realised that this subsampling is a choice of the deeplab designers...
